app/api/guests+api.ts

```diff
@@ -0,0 +1,69 @@
+import { createGuestInsecure, getGuestsInsecure } from '../../database/guests';
+import { ExpoApiResponse } from '../../ExpoApiResponse';
+import {
+  type Guest,
+  guestsSchema,
+} from '../../migrations/00000-createTableGuests';
+
+type GuestResponseBodyPost =
+  | { guest: Guest }
+  | { error: string; errorIssues?: { message: string }[] };
+
+export async function POST(
+  request: Request,
+): Promise<ExpoApiResponse<GuestResponseBodyPost>> {
+  const requestBody = await request.json();
+
+  const result = guestsSchema.safeParse(requestBody);
+
+  if (!result.success) {
+    return ExpoApiResponse.json(
+      {
+        error: 'Request does not contain guest object',
+        errorIssues: result.error.issues,
+      },
+      {
+        status: 400,
+      },
+    );
+  }
+
+  const newGuest = {
+    firstName: result.data.firstName,
+    lastName: result.data.lastName,
+    attending: result.data.attending,
+  };
+
+  const guest = await createGuestInsecure(newGuest);
+
+  if (!guest) {
+    return ExpoApiResponse.json(
+      { error: 'Guest not created' },
+      {
+        status: 500,
+      },
+    );
+  }
+
+  return ExpoApiResponse.json({ guest: guest });
+}
+
+type GuestResponseBodyGet = { guests: Guest[] };
```

Comment:
this should also not be on 1 line, our type aliases which are objects are formatted to multiple lines, even with only 1 property